data/students.json

```diff
@@ -1,75 +1,314 @@
+      "displayName": "Josh White",
+      "slug": "white",
+      "year": 2015,
+      "thesisPages": 20,
+      "thesisTitle": "Bad Year",
+      "digitalTitle": "I Went on a Walk and I Saw a Snake",
+      "digitalUrl": "15white/white.html",
+      "thesisPath": "15white/"
+    },
+    {
+      "displayName": "Shane McDermott",
+      "slug": "mcdermott",
+      "year": 2015,
+      "thesisPages": 14,
+      "thesisTitle": "The Seahorse Adventures",
+      "digitalTitle": "",
+      "digitalUrl": "",
+      "thesisPath": "15mcdermott/"
+    },
+    {
+      "displayName": "Ryan Bitinis",
+      "slug": "bitinis",
+      "year": 2016,
+      "thesisPages": 10,
+      "thesisTitle": "The Tale of Old Ryan",
+      "digitalTitle": "This Will All End in Flames",
+      "digitalUrl": "16bitinis/comics.html",
+      "thesisPath": "16bitinis/"
+    },
+    {
+      "displayName": "Kayan Cheung-Miaw",
+      "slug": "cheungmiaw",
+      "year": 2016,
+      "thesisPages": 21,
+      "thesisTitle": "Your Struggle is Your Strength",
+      "digitalTitle": "The Experience of Many Lifetimes",
+      "digitalUrl": "16cheungmiaw/kayan_cover.html",
+      "thesisPath": "16cheungmiaw/"
+    },
+    {
+      "displayName": "Peter Glanting",
+      "slug": "glanting",
+      "year": 2016,
+      "thesisPages": 15,
+      "thesisTitle": "Tomorrow Jerks #2",
+      "digitalTitle": "Boardsock Empire",
+      "digitalUrl": "16glanting/static-sock-town-a.html",
+      "thesisPath": "16glanting/"
+    },
+    {
+      "displayName": "Joseph Hewitt",
+      "slug": "hewitt",
+      "year": 2016,
+      "thesisPages": 24,
+      "thesisTitle": "How to Be Human, Day One",
+      "digitalTitle": "Ghost Radio",
+      "digitalUrl": "16hewitt/",
+      "thesisPath": "16hewitt/"
+    },
+    {
+      "displayName": "Masami Inoue",
+      "slug": "inoue",
+      "year": 2016,
+      "thesisPages": 15,
+      "thesisTitle": "Whale and Albatross",
+      "digitalTitle": "Death",
+      "digitalUrl": "16inoue/home.html",
+      "thesisPath": "16inoue/"
+    },
+    {
+      "displayName": "Cordelia McGee-Tubb",
+      "slug": "mcgeetubb",
+      "year": 2016,
+      "thesisPages": 18,
+      "thesisTitle": "Selected Stories from Your Hands Are Clean and You Have Nothing to Worry About",
+      "digitalTitle": "Brocedurally Generated",
+      "digitalUrl": "16mcgeetubb/",
+      "thesisPath": "16mcgeetubb/"
+    },
+    {
+      "displayName": "Paul Morehead",
+      "slug": "morehead",
+      "year": 2016,
+      "thesisPages": 18,
+      "thesisTitle": "The Free Radicals",
+      "digitalTitle": "James and the Oak",
+      "digitalUrl": "16morehead/James_and_the_Oakp1.html",
+      "thesisPath": "16morehead/"
+    },
+    {
+      "displayName": "Kristin Olson",
+      "slug": "olson",
+      "year": 2016,
+      "thesisPages": 6,
+      "thesisTitle": "Sick Bed Blues",
+      "digitalTitle": "The Crystal Ball",
+      "digitalUrl": "16olson/pg1.html",
```

Comment:
Why are some digitalUrls directories and others files?

Reply:
this is Q i am most interested in.

Reply:
when there's an index.html file i just linked to the directory...if that's bad form i can just paste in the "index.html" that's nbd